applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.cpp

```diff
@@ -28,6 +28,29 @@
 
```

Comment:
This file has lots of aesthetic changes. Please remove them, otherwise it becomes very difficult to track the real changes.